### PR TITLE
Remove pages-3.1 feature from facesContainer-4.0 feature

### PR DESCRIPTION
**Pull request by pnicolucci**
for #22202

### CODE REVIEW

**Comment by pnicolucci** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsfContainer_fat_2.3

**Comment by pnicolucci** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsfContainer_fat_2.3

File: dev/com.ibm.websphere.appserver.features/visibility/public/facesContainer-4.0/io.openliberty.facesContainer-4.0.feature [hunk 1]
```diff
@@ -17,7 +17,7 @@ IBM-API-Package: org.jboss.weld;type="internal",\
   com.ibm.websphere.appserver.servlet-6.0, \
   com.ibm.websphere.appserver.eeCompatible-10.0, \
   io.openliberty.jakarta.validation-3.0, \
-  io.openliberty.pages-3.1, \
+  io.openliberty.expressionLanguage-5.0, \
   com.ibm.websphere.appserver.jndi-1.0, \
```
Comment (pnicolucci) [inline dev/com.ibm.websphere.appserver.features/visibility/public/facesContainer-4.0/io.openliberty.facesContainer-4.0.feature:21]: pages-3.1 pulls in expressionLanguage-5.0 so we need to explicitly pull in expressionLanguage-5.0 here.

**Comment by pnicolucci** [comment]
#libby

**Review by volosied — APPROVED** [review]

**Pull request merged**